--- a/partitionmanager.cpp	2026-07-26 07:48:47.550989300 +0300
+++ b/partitionmanager.cpp	2026-07-26 07:48:48.061886900 +0300
@@ -1872,7 +1872,7 @@
 	std::vector<TWPartition*>::iterator iter;
 	int data_size = 0;

-	gui_msg("update_part_details=Updating partition details...");
+	// gui_msg("update_part_details=Updating partition details...");
 	for (iter = Partitions.begin(); iter != Partitions.end(); iter++) {
 		(*iter)->Update_Size(true);
 		if ((*iter)->Can_Be_Mounted) {
@@ -1933,14 +1933,14 @@
 			}
 		}
 	}
-	gui_msg("update_part_details_done=...done");
+	// gui_msg("update_part_details_done=...done");
 	DataManager::SetValue(TW_BACKUP_DATA_SIZE, data_size);
 	string current_storage_path = DataManager::GetCurrentStoragePath();
 	TWPartition* FreeStorage = Find_Partition_By_Path(current_storage_path);
 	if (FreeStorage != NULL) {
 		// Attempt to mount storage
 		if (!FreeStorage->Mount(false)) {
-			gui_msg(Msg(msg::kWarning, "unable_to_mount_storage=Unable to mount storage"));
+			// gui_msg(Msg(msg::kWarning, "unable_to_mount_storage=Unable to mount storage"));
 			DataManager::SetValue(TW_STORAGE_FREE_SIZE, 0);
 		} else {
 			DataManager::SetValue(TW_STORAGE_FREE_SIZE, (int)(FreeStorage->Free / 1048576LLU));
--- a/twrp-functions.cpp	2026-07-26 07:48:47.675504600 +0300
+++ b/twrp-functions.cpp	2026-07-26 07:48:48.133402400 +0300
@@ -665,7 +665,7 @@
 	// Check for and run startup script if script exists
 	struct stat st;
 	if (stat(script_file, &st) == 0) {
-		gui_msg(Msg("run_script=Running {1} script...")(display_name));
+		// gui_msg(Msg("run_script=Running {1} script...")(display_name));
 		chmod(script_file, S_IRWXU | S_IRGRP | S_IXGRP | S_IROTH | S_IXOTH);
 		TWFunc::Exec_Cmd(script_file);
 		gui_msg("done=Done.");
@@ -1305,7 +1305,7 @@
 			gui_warn("no_kernel_selinux=Kernel does not have support for reading SELinux contexts.");
 		} else {
 			free(contexts);
-			gui_msg("full_selinux=Full SELinux support is present.");
+			// gui_msg("full_selinux=Full SELinux support is present.");
 		}
 	}
 }
--- a/twrp.cpp	2026-07-26 07:48:47.799356500 +0300
+++ b/twrp.cpp	2026-07-26 07:48:48.182402200 +0300
@@ -112,7 +112,7 @@
 			PartitionManager.Unmap_Super_Devices();
 		}

-		gui_msg(Msg("fastboot_console_msg=Entered Fastboot mode..."));
+		// gui_msg(Msg("fastboot_console_msg=Entered Fastboot mode..."));
 		// Check for and run startup script if script exists
 		TWFunc::check_and_run_script("/system/bin/runatboot.sh", "boot");
 		TWFunc::check_and_run_script("/system/bin/postfastboot.sh", "fastboot");
@@ -262,7 +262,7 @@
 			gui_msg("mtp_enabled=MTP Enabled");
 		property_set("mtp.crash_check", "0");
 	} else if (strcmp(mtp_crash_check, "0")) {
-		gui_warn("mtp_crash=MTP Crashed, not starting MTP on boot.");
+		// gui_warn("mtp_crash=MTP Crashed, not starting MTP on boot.");
 		DataManager::SetValue("tw_mtp_enabled", 0);
 		PartitionManager.Disable_MTP();
 	} else if (crash_counter == 1) {
--- a/gui/action.cpp	2026-07-26 07:48:47.929661300 +0300
+++ b/gui/action.cpp	2026-07-26 07:48:48.229916200 +0300
@@ -1053,7 +1053,7 @@
 				string injectcmd = "injecttwrp --dump /tmp/backup_recovery_ramdisk.img /tmp/injected_boot.img --flash bd=" + Boot->Actual_Block_Device;
 				TWFunc::Exec_Cmd(injectcmd);
 			}
-			gui_msg("done=Done.");
+			// gui_msg("done=Done.");
 		}
 	}
 }
@@ -1485,7 +1485,7 @@
 		simulate_progress_bar();
 	} else {
 		TWFunc::Exec_Cmd("injecttwrp --dump /tmp/backup_recovery_ramdisk.img /tmp/injected_boot.img --flash");
-		gui_msg("done=Done.");
+		// gui_msg("done=Done.");
 	}

 	operation_end(op_status);
