--- a/gui/action.cpp	2026-07-26 07:48:47.929661300 +0300
+++ b/gui/action.cpp	2026-07-26 07:48:48.229916200 +0300
@@ -1449,7 +1449,7 @@
 		string injectcmd = "injecttwrp --dump /tmp/backup_recovery_ramdisk.img /tmp/injected_boot.img --flash bd=" + Boot->Actual_Block_Device;
 	      TWFunc::Exec_Cmd(injectcmd);
 	    }
-	  gui_msg("done=Done.");
+	  // gui_msg("done=Done.");
 	}
     }
 }
@@ -2029,7 +2029,7 @@
     {
       TWFunc::
 	Exec_Cmd
 	("injecttwrp --dump /tmp/backup_recovery_ramdisk.img /tmp/injected_boot.img --flash");
-      gui_msg("done=Done.");
+      // gui_msg("done=Done.");
     }
 
   operation_end(op_status);
